src/database/migrations/2022_05_19_071616_create_likes_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateLikesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('likes', function (Blueprint $table) {
+            $table->id();
+            $table->timestamps();
+            $table->unsignedInteger('user_id');
+            $table->unsignedBigInteger('movie_id');
```

Comment:
Nemas foreignId constraint ovde
Za svaki id iz druge tabele moras foreig key da ubacis jel ovako mozemo proslediti neki bezveze id i to ce proci